index.php

```diff
@@ -10,153 +10,201 @@
+} else {
+    $results_per_page = 5;
+}
+
+if (isset($_GET['search'])) {
+    $search = htmlspecialchars($_GET['search']);
+    if (isset($_GET['find'])) {
+        $find = $_GET['find'];
+    } else {
+        $find = 1;
+    }
+    $squery = "SELECT * FROM `user_table` WHERE `id` LIKE '%" . $search . "%' or `firstname` LIKE '%" . $search . "%' or `lastname` LIKE '%" . $search . "%' or `age` LIKE '%" . $search . "%' or `email` LIKE '%" . $search . "%' or `gender` LIKE '%" . $search . "%' or `occupation` LIKE '%" . $search . "%'";
+    $searchresult = mysqli_query($conn, $squery)  or die(mysqli_error($conn));
+    $search_result = mysqli_num_rows($searchresult);
+    $slastpage = ceil($search_result / $results_per_page);
+    $search_first_result = ($find - 1) * $results_per_page;
+    if ($find > $slastpage) {
+        $find = $slastpage;
+    }
+    if ($find < 1) {
+        $find = 1;
+    }
+} else {
+    $query = "select * from user_table";
+    $result = mysqli_query($conn, $query);
+    $number_of_result = mysqli_num_rows($result);
+    //determine the total number of pages available  
+    $lastpage = ceil($number_of_result / $results_per_page);
+    //determine which page number visitor is currently on  
+    if (isset($_GET['page'])) {
+        $page = $_GET['page'];
+    } else {
+        $page = 1;
+    }
+    $page_first_result = ($page - 1) * $results_per_page;
+}
 ?>
-<div class="container-fluid mb-4">
+<div class="container">
     <div class="header mt-0 mb-3">
         <h2>Home Page</h2>
     </div>
-    <div class="d-flex justify-content-around">
-        <form action="<?php echo $_SERVER['PHP_SELF']; ?>" method="get" id="myFormId">
-            <div class="row  ml-4 mr-4 p-3">
-                <div class="col-sm-5">
-                    <input type="text" name="search" value="<?php if (isset($_GET['search'])) {
-                                                                echo $_GET['search'];
-                                                            } ?>" autocomplete="off">
-                    <button type="submit" class="btm btn-success">Search</button>
+    <form action="<?php echo $_SERVER['PHP_SELF']; ?>" method="get" id="myFormId">
+        <div class="form-group">
+            <div class="row  ml-2 mr-2 p-2">
+                <div class="col-sm-6">
+                    <input type="text" class="form-control" name="search" value="<?php if (isset($_GET['search'])) {
+                                                                                        echo $_GET['search'];
+                                                                                    } ?>" autocomplete="off" placeholder="search....." required>
+                </div>
+                <div class="col-sm-6">
+                    <label for="num_rows">Number of rows</label>
+                    <select name="results" required>
+                        <option value="" disabled selected>Choose option</option>
```

Comment:
Can you do create the below list via loop ? Also, set the selected value dynamically. As there would be more options in future so we don't have time to alter the code. so make it dynamic via loop.